flameCalculator/main.js

```diff
@@ -392,45 +418,14 @@ function getProbability(item_level, flame_type, item_type, desired_stat, non_adv
     //attack_tier
 
     var solutions = []
-    var upper_limit = 7
-    var sec_upper_limit = 7
-    var combo_four_upper_limit = 7
-    var combo_five_upper_limit = 7
-    var att_upper_limit = 7
-    var all_stat_upper_limit = 7
-
-    var lower_limit = 3
-
-    if (non_advantaged_item) {
```

Comment:
This code here is why the bug never happened in prod. It skipped flame tiers that would never be possible to get so options with 0 probability didn't show up in `getTierProbability`. At first my changes didn't update these numbers to work with the new flame types (only <master fusion was causing trouble since it's the only one that can't have tier 6) so I ran into the bug. 

Later I updated this code via the helpers above so that it would work with any flame type because not using it decreased performance quite a bit (since the algorithm is like O(n^8) or whatever).